### PR TITLE
Port interface-app Arlo on-change subscription code to common-app

### PR DESCRIPTION
**Pull request by amrutasali**
Port interface-app Arlo on-change subscription code to common-app since transformer infra support for subscription is post buzznik

### CODE REVIEW

File: src/translib/common_app.go [hunk 1]
```diff
@@ -726,6 +733,44 @@ func checkAndProcessLeafList(existingEntry db.Value, tblRw db.Value, opcode int,
...
+			log.Errorf("Subscribe not supported for %s!", pathInfo.Template)
+			return nil, nil, notSupported
+		}
+		ifName := pathInfo.Var("name")
+		if len(ifName) == 0 {
+			return nil, nil, errors.New("Empty Interface name")
+		}
+		log.Info("Interface name = ", ifName)
+		ifKey := db.Key{Comp: []string{ifName}}
+		portTblTs := &db.TableSpec{Name: "PORT_TABLE"}
```
Comment (Tejaswi-Goel) [inline src/translib/common_app.go:755]: PORT_TABLE is only for physical interfaces, to make it more generic(subscribe support for other interface types-LAG, VLAN..)  we will have to find type of interface from ifName and then  look into its table in appDB.

Comment (amrutasali) [reply src/translib/common_app.go:755]: @Tejaswi-Goel  - we were just asked to pot the  code from arlo....no new support for now

**Comment by sameerdell** [comment]
decided to do it in infra, closing pull request.

**Comment by sameerdell** [comment]
decided to do it in infra, closing pull request.

**Pull request closed without merging**